step.yml

```diff
@@ -51,300 +49,218 @@ toolkit:
+      - `up_until_maximum_repetitions`: Tests will repeat up until maximum repetitions.
+
+      The input value together with Maximum Test Repetitions (`maximum_test_repetitions`) input sets xcodebuild’s `-run-tests-until-failure` / `-retry-tests-on-failure` or `-test-iterations option`.
+    value_options:
+    - "none"
+    - "until_failure"
+    - "retry_on_failure"
+    - "up_until_maximum_repetitions"
+
+- maximum_test_repetitions: 3
+  opts:
+    category: Test Repetition
+    title: Maximum Test Repetitions (Available in Xcode 13+)
+    summary: The maximum number of times a test repeats based on the Test Repetition Mode (`test_repetition_mode`).
+    description: |-
+      The maximum number of times a test repeats based on the Test Repetition Mode (`test_repetition_mode`).
+
+      Should be more than 1 if the Test Repetition Mode is other than `none`.
+
+      The input value sets xcodebuild’s `-test-iterations` option.
+    is_required: true
+
+- relaunch_tests_for_each_repetition: "no"
+  opts:
+    title: Relaunch Tests for Each Repetition (Available in Xcode 13+)
+    category: Test Repetition
+    summary: If this input is set, tests will launch in a new process for each repetition.
+    description: |-
+      If this input is set, tests will launch in a new process for each repetition.
+
+      By default, tests launch in the same process for each repetition.
+
+      The input value sets xcodebuild’s `-test-repetition-relaunch-enabled` option.
+    value_options:
+    - "yes"
+    - "no"
+
+- should_retry_test_on_fail: "no"
+  opts:
+    category: Test Repetition
+    title: Should retry tests on failure? (Not available in Xcode 13+)
+    summary: If set, the Step will rerun the tests in the case of failed tests.
+    description: |-
+      If this input is set, the Step will rerun the tests in the case of failed tests.
+
+      Note that all the tests will be rerun, not just the ones that failed.
+
+      This input is not available if you are using Xcode 13+. In that case, we recommend using the `retry_on_failure` Test Repetition Mode (`test_repetition_mode`).
+    value_options:
+    - "yes"
+    - "no"
+    is_required: true
+
+# xcodebuild configuration
+
+- xcconfig_content: "COMPILER_INDEX_STORE_ENABLE = NO"
+  opts:
+    category: xcodebuild configuration
+    title: Build settings
+    summary: Apply the build settings as overrides.
+    description: |-
+      Build settings to override the project's build settings.
+
+      The input value sets xcodebuild’s `-xcconfig` option.
+
+- perform_clean_action: "no"
+  opts:
+    category: xcodebuild configuration
+    title: Perform clean action
+    summary: If this input is set, `clean` xcodebuild action will be performed besides the `test` action.
+    value_options:
+    - "yes"
+    - "no"
+    is_required: true
+
+- xcodebuild_options: ""
+  opts:
+    category: xcodebuild configuration
+    title: Additional options for the xcodebuild command
```

Comment:
`xcodebuild`

Reply:
also in summary and description

Reply:
I thought we don't intend to monospace `xcodebuild` based on the input restructure document. Because there are other usages as well.

I think we should discuss this offline.